website/src/graphql/dailp/queries.graphql

```diff
@@ -603,10 +594,16 @@ mutation PostComment($input: PostCommentInput!) {
     ... on AnnotatedForm {
       __typename
       id
+      comments {
+        ...CommentFields
+      }
     }
     ... on DocumentParagraph {
       __typename
       id
+      comments {
+        ...CommentFields
+      }
```

Comment:
We fetch the comments and the fields we need when we post, so we have an up-to-date version on the frontend as soon as the mutation finishes.